frontend/mempool-frontend-config.sample.json

```diff
@@ -12,6 +12,7 @@
   "NGINX_PORT": "80",
   "BLOCK_WEIGHT_UNITS": 4000000,
   "MEMPOOL_BLOCKS_AMOUNT": 8,
+  "INDEXING_BLOCKS_AMOUNT": 432,
```

Comment:
Shouldn't this be 1100?

Reply:
Actually, wrong file. It should be in backend only.